src/guacd/systemd/guacd.service.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Do systemd unit files support comments? Assuming they do, this new file should include the usual license boilerplate.

Reply:
Probably so...will add it.

Reply:
Added.